### PR TITLE
Update scorecard section text

### PR DESCRIPTION
**Pull request by MonikaFu**
Updating text in executive summary scorecard section according to analysts' specifications.

Testing the output here: https://github.com/RMI-PACTA/workflow.transition.monitor/pull/344

NOTE: the .Rproj is also updated as it seems to be modified when Antoine made a PR to this repo. I want to get it back to the shape that it is in on my computer since I am the maintainer.

### CODE REVIEW

File: pacta.executive.summary.Rproj [hunk 1]
```diff
@@ -20,6 +20,7 @@ BuildType: Package
 PackageUseDevtools: Yes
 PackageInstallArgs: --no-multiarch --with-keep.source
 PackageRoxygenize: rd,collate,namespace
+
```
Comment (cjyetman) [inline pacta.executive.summary.Rproj:23]:
```suggestion
```

**Review by cjyetman — APPROVED** [review]

File: pacta.executive.summary.Rproj (hunk 1 above)
Comment (MonikaFu) [reply pacta.executive.summary.Rproj:23]: @cjyetman I don't know why but RStudio keeps on changing it on my machine and adding the space..

Comment (cjyetman) [reply pacta.executive.summary.Rproj:23]: whoops, I though @Antoine-Lalechere's machine added the space, so I suggested removing it 🤷🏻

**Review by jdhoffa — APPROVED** [review]

File: pacta.executive.summary.Rproj (hunk 1 above)
Comment (MonikaFu) [reply pacta.executive.summary.Rproj:23]: no, I guess it was the other way around

**Pull request merged**